cpp/src/arrow/compute/exec/options.h

```diff
@@ -438,7 +450,5 @@ class ARROW_EXPORT TableSinkNodeOptions : public ExecNodeOptions {
   std::shared_ptr<Table>* output_table;
 };
 
-/// @}
```

Comment:
You shouldn't remove this, this matches the opening brace in `\addtogroup execnode-options` above.